docs/admin/disruptions.md

```diff
@@ -34,11 +34,13 @@ disruption budget to be violated.
 
 ## Specifying a PodDisruptionBudget
 
-A `PodDisruptionBudget` has two components: a selector to specify the set of
-pods, and a description of the minimum number of available pods for a disruption
-to be allowed.  The latter can be either an absolute number or a percentage.  In
-typical usage, a single budget would be used for a collection of pods managed by
-a controller—for example, the pods in a single ReplicaSet.
+Currently a `PodDisruptionBudget` has two components: a selector to specify the
```

Comment:
"below which eviction is not allowed" is brief but incorrect in the way we just fixed.  We won't allow an eviction that would *bring* the number of available pods below this threshold.

I can't think of a succinct way to put this.

(added the comma)